pkg/sdn/plugin/subnets.go

```diff
@@ -107,6 +107,17 @@ func (master *OsdnMaster) deleteNode(nodeName string) error {
 	}
 
 	log.Infof("Deleted HostSubnet %s", hostSubnetToString(sub))
+
+	_, sn, err := net.ParseCIDR(sub.Subnet)
+	if err != nil {
+		return fmt.Errorf("error parsing network address %q: %v", sub.Subnet, err)
+	} else {
+		err := master.subnetAllocator.ReleaseNetwork(sn)
```

Comment:
if this fails, is this function re-entrant? if not, how does cleanup occur in that case?